### PR TITLE
Android builder support for server

### PR DESCRIPTION
**Pull request by soulchips**
# Description
Adding docs on how server customers can use Android builders

# Reasons
https://circleci.atlassian.net/browse/SERVER-1251

### CODE REVIEW

File: jekyll/_cci2/server-3-install-build-services.adoc [hunk 1]
```diff
@@ -380,6 +380,15 @@ Once you have configured the environmental variables and contexts, rerun the rea
...
 
+==== Enabling Nomad Client for External Image Repository
+For enabling Nomad Client to download images from External or Private repo, You can follow below step -
+
+* Make Image Registry Public to Private/Internal Network so that Nomad client can download them without any authentication.
+* Or, Embed the Private Image repository credentials in Nomad Client Image at `/root/.docker/config.json` assuming credentials are static.
+
+NOTE: If you are using AWS, GCP or any other cloud image registry, most of them are using temporary docker credentials valid only for few hours to use. As soon as those credentials are expired, your nomad client will be unable to download the image from private image repository if image is not existing in Nomad client.
+
+
```
Comment (ryanwohara) [inline jekyll/_cci2/server-3-install-build-services.adoc:391]: This seems to have snuck in

Comment (soulchips) [reply jekyll/_cci2/server-3-install-build-services.adoc]: sorted

**Review by atulsingh0 — APPROVED** [review]

**Review by ryanwohara — APPROVED** [review]
Nice work on this

**Comment by devinmarieb** [comment]
@rosieyohannan - Final review is yours. I made some small edits for consistency.

**Comment by devinmarieb** [comment]
@rosieyohannan - Final review is yours. I made some small edits for consistency.

**Pull request merged**